src/Features/CSharp/Portable/Diagnostics/Analyzers/CSharpTypeStyleDiagnosticAnalyzerBase.cs

```diff
@@ -26,17 +20,6 @@ internal abstract partial class CSharpTypeStyleDiagnosticAnalyzerBase :
 
         public override DiagnosticAnalyzerCategory GetAnalyzerCategory() => DiagnosticAnalyzerCategory.SemanticSpanAnalysis;
 
-        public override bool OpenFileOnly(Workspace workspace)
-        {
```

Comment:
code style analyzer is weird one I believe, since engine is specifically rewritten to ignore code style analyzer's descriptor. it is special ability grant to only IDE code style analyzer which we want to fix at some point. since they return dynamic descriptor that are not allowed to any other analyzers.

basically they can return diagnostic that is not from descriptor they return.

so, some of these analyzer used to not run on OOP when this return true. will all these run on OOP now? or never. I think code style analyzer always return fake hidden descriptor? (since it doesn't matter for them)